action.yml

```diff
@@ -0,0 +1,106 @@
+name: Silverbackltd Build Action
+author: 'ApeWorX Ltd.'
+description: 'A GitHub Action to build docker images for a silverback repository and push to a container registry'
+branding:
+  icon: 'check-circle'
+  color: 'green'
+
+inputs:
+  push:
+    description: 'Push the image to ghcr'
+    required: false
+    default: 'false'
+  tag:
+    description: 'Tag of the silverback docker images'
+    required: false
+    default: 'latest'
+  registry:
+    description: 'The registry location you want to push to.'
+    required: false
+    default: 'ghcr.io'
+  username:
+    description: 'Add your username for your registry'
+    required: false
+    default: ${{ github.actor }}
```

Comment:
Does this work?

Reply:
need to test an actual push, I doubt it. I should probably `default: ''` like I did with password, honestly. We don't need it unless we push